src/splatting/Splatting.js

```diff
@@ -196,13 +207,15 @@ function ApplyTransMatrix(sourceMatrix, transMatrix, scaleF)
 function createWorker(self) {
     let buffer;
     let vertexCount = 0;
+    let doneLoading = false;
     let viewProj;
     // 6*4 + 4 + 4 = 8*4
     // XYZ - Position (Float32)
     // XYZ - Scale (Float32)
     // RGBA - colors (uint8)
     // IJKL - quaternion/rot (uint8)
```

Comment:
should update this comment with documentation about the cluster label (4 uint8s?)